zilliqa/tests/it/consensus.rs

```diff
@@ -1,11 +1,138 @@
 use crate::CombinedJson;
+use crate::Network;
 use ethabi::Token;
-
+use ethers::{providers::Middleware, types::TransactionRequest};
+use tracing::*;
 use zilliqa::state::Address;
 
-use crate::Network;
+// Test that a node that joins later can sync up to the latest block and become a
```

Comment:
These commented out blocks should probably be cleaned up before merging (including the entire commented test)